test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -110,7 +145,8 @@ private fun InstrumentationTestContext.calculateDummyShards(
 
 @VisibleForTesting
 internal fun InstrumentationTestContext.getFlankTestMethods(
-    testFilter: TestFilter
+    testFilter: TestFilter,
+    parameterizedTests: String = FlankDefaults.DEFAULT_PARAMETERIZED_TESTS
```

Comment:
I think the default value here is redundant. I could not find a method call without `parameterizedTests`

Reply:
👍

Reply:
This will stay as there are multiple tests that rely upon it.

Reply:
If it must, let it be. It's just a bit smelly ( 😿  look, you made a kitty sad)